nuxeo-retention-core/src/main/resources/OSGI-INF/retention-listener-contrib.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.retention.listeners">
+
+  <extension target="org.nuxeo.ecm.core.event.EventServiceComponent"
+    point="listener">
+    <listener name="retentionRecordCheckerListener"
+      class="org.nuxeo.ecm.retention.listener.RetentionRecordCheckerListener">
+      <event>checkRecordEvent</event> 
```

Comment:
Idea was to listen to every event and filter on only Record document; cause we don't know which event will be required to enter in retention, or to just pass to the next retention's state